libs/features/mas/mas/src/mas.js

```diff
@@ -19,5 +18,7 @@ const config = () => ({
 init(config);
 
 if (features.includes('merch-card')) {
-    import(`${origin}/libs/deps/merch-card-all.js`);
+  import('../../web-components/src/merch-card.js');
+  import('../../web-components/src/merch-icon.js');
+  import('../../web-components/src/merch-datasource.js');
```

Comment:
sorry to be a pain but i wonder if this shouldn't be wrapped with Promise.allSettled + await (await was not done l22 before but i think that was a mistake that _could_ have generated some issues)

Reply:
Added